bamboo/bootstrap-unit-tests.sh

```diff
@@ -14,6 +14,12 @@ docker_command="docker exec -t ${container_id}-build_env-1 /bin/bash -c"
 
 docker ps -a
 
+echo 'Removing running docker containers...'
+
+docker rm -f $(docker ps -q) || true
```

Comment:
This is a pretty broad `rm` command, no?

Reply:
It looks like this is from a separate PR. This should either be removed from this one or https://github.com/nasa/cumulus/pull/3586 should be merged first.

In any case, I _think_ this is appropriate as we're saying effectively 'there should be no running docker containers'. We're clearing the table and then starting all of the appropriate containers (e.g. buid env, ftp server, ElasticSearch, etc.).

Reply:
Ah I didn't see the comment above 😆 https://github.com/nasa/cumulus/pull/3583/files#r1488189445